tests/testthat/_snaps/error-call.md

```diff
@@ -0,0 +1,175 @@
+# failing common type reports correct error call
+
+    Code
+      (expect_error(my_function()))
+    Output
+      <error/vctrs_error_incompatible_type>
+      Error:
```

Comment:
Why doesn't it say `in my_function():` here? There are a few below where I expected that

Reply:
oh yes these still need to be fixed. It's already an improvement since it no longer shows stopper calls.

Reply:
This is now better, thanks!